app/src/main/assets/timetable/cs1sem.json

```diff
@@ -1,37 +1,152 @@
 {
   "Monday": {
-    "9:00-10:00": "CS B.Tech sem 1",
-    "10:00-11:00": "Lorem Ipsum",
-    "11:15-12:15": "Lorem Ipsum",
-    "12:15-1:15": "Lorem Ipsum",
-    "3:00-6:00": "Lorem Ipsum"
+    "9:00-10:00": {
```

Comment:
@Retarded-Ape No need to add staff key-value pair. Just fill subject name in the time key like this
"9:00-10:00": "SM130",
"10:00-11:00": "FEE132", and so on.